src/converters/app-converter.ts

```diff
@@ -83,7 +83,7 @@ export class AppConverter {
 
         const stackConverter = this.stacks.get(artifact.id);
         if (!stackConverter) {
-            throw new Error(`missing CDK Stack for artifact ${artifact.id}`);
+            throw new Error(`[CDK Adapter] missing CDK Stack for artifact ${artifact.id}`);
```

Comment:
Is it worth creating a custom error class that sets this?

Reply:
Yeah good idea